README.md

```diff
@@ -1 +1,37 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+
+Hello Microverse Project
+
+<i>Built With HTML5, CSS3, GIT, Bootstrap, SASS and VSCODE</i>
+
+Live Demo : [The-livedemo-link]()
+
+<h1>Getting Started</h1>
+
+To get a local copy up and running follow these simple example steps.
+
+1. Go to https://karvel-code.github.io/Microverse-Capstone/index.html
+2. Click on code and copy the html code.
+3. Use git bash to clone this repository on your local machine.
+4. Run git checkout -b your-branch-name Make your contributions
+5. Push your branch up to your forked repository
+6. Open a Pull Request with a detailed description to the development branch of the original project for a review
+
+Prerequisites Text editor, Github profile and Git.
+
+<h2>Author</h2>
```

Comment:
Please change the <h2> tag into an <h1> tag. It is important to have an <h1> tag on all your html pages for accessibility purposes.